infrastructures/infrastructure-gce/src/main/java/org/ow2/proactive/resourcemanager/nodesource/infrastructure/GCEInfrastructure.java

```diff
@@ -200,49 +200,49 @@ private void validate(Object[] parameters) {
         if (parameters[parameterIndex] == null) {
             parameters[parameterIndex] = "";
         }
-        // rmHostname
+        // image
         parameterIndex++;
         if (parameters[parameterIndex] == null) {
-            throw new IllegalArgumentException("The resource manager hostname must be specified");
-        }
-        if (parameters[parameterIndex].toString().contains("/")) {
-            throw new IllegalArgumentException(String.format("Invalid hostname %s (hostname should not contains '/').",
-                                                             parameters[parameterIndex]));
+            parameters[parameterIndex] = DEFAULT_IMAGE;
         }
-        // connectorIaasURL
+        // region
         parameterIndex++;
         if (parameters[parameterIndex] == null) {
-            throw new IllegalArgumentException("The connector-iaas URL must be specified");
+            parameters[parameterIndex] = DEFAULT_REGION;
         }
-        // downloadCommand
+        // ram
         parameterIndex++;
         if (parameters[parameterIndex] == null) {
-            throw new IllegalArgumentException("The command for downloading the node jar must be specified");
+            parameters[parameterIndex] = String.valueOf(DEFAULT_RAM);
         }
-        // additionalProperties
+        // cores
         parameterIndex++;
         if (parameters[parameterIndex] == null) {
-            parameters[parameterIndex] = "";
+            parameters[parameterIndex] = String.valueOf(DEFAULT_CORES);
         }
-        // image
+        // rmHostname
         parameterIndex++;
         if (parameters[parameterIndex] == null) {
-            parameters[parameterIndex] = DEFAULT_IMAGE;
+            throw new IllegalArgumentException("The resource manager hostname must be specified");
         }
-        // region
+        if (parameters[parameterIndex].toString().contains("/")) {
+            throw new IllegalArgumentException(String.format("Invalid hostname %s (hostname should not contains '/').",
+                                                             parameters[parameterIndex]));
+        }
+        // connectorIaasURL
         parameterIndex++;
         if (parameters[parameterIndex] == null) {
-            parameters[parameterIndex] = DEFAULT_REGION;
+            throw new IllegalArgumentException("The connector-iaas URL must be specified");
         }
-        // ram
+        // nodeJarUrl
         parameterIndex++;
         if (parameters[parameterIndex] == null) {
-            parameters[parameterIndex] = String.valueOf(DEFAULT_RAM);
+            throw new IllegalArgumentException("The url for downloading the node jar must be specified");
```

Comment:
Be consistent in your abbreviations. In the upper exception, you use `URL` and here `url`. I think using `URL` is better in this case.

Reply:
Done.